### ISSUE TITLE
Label background colouring

### ISSUE THREAD

**Issue by martingraham** [arose/ngl#444]
Is there a way of setting background colours on a per-label basis for a LabelRepresentation e.g. as an array of colours similar to the array of labels? It'd be nice to give my chain labels the same colour as the chain representations themselves (I can see how to work out the colours from another issue #247  )

**Comment by arose** [comment]
No, that is not possible. The 'Annotation' feature gives you more control over the appearance (at the expense of performance). I just (ed58d1bda3776355989b5659df6a45a1e610a553) improved it a bit to handle your use case. See https://codepen.io/arose/pen/GQqGJv?&editors=001.
